packages/core/saved-objects/docs/openapi/components/headers/kbn_xsrf.yaml

```
@@ -3,4 +3,8 @@ schema:
 in: header
 name: kbn-xsrf
 description: Cross-site request forgery protection
+<<<<<<< HEAD
 required: true
+=======
+required: true
+>>>>>>> 6061fa36a2b (passed validation)
```

Mind this unresolved conflict:
```suggestion
required: true
```

---

Oh thank you for catching these!